Isodose/CMakeLists.txt

```diff
@@ -42,9 +46,12 @@ set(MODULE_UI_SRCS
 )
 
 set(MODULE_TARGET_LIBRARIES
+  vtkSlicer${MODULE_NAME}ModuleMRML
   vtkSlicer${MODULE_NAME}ModuleLogic
-  vtkSlicer${MODULE_NAME}ModuleWidgets
+#  vtkSlicer${MODULE_NAME}ModuleWidgets

```

Comment:
Why was it needed to comment this? The include directlry is still added, so removing the lib is inconsistent.